### DISCUSSION TITLE
How-To add a shell script on a blog (needs copy and paste) into brew?

### DISCUSSION THREAD

**Discussion by GETandSELECT** [Homebrew/discussions#601] [Everyday usage] [answered]
Hi

I would like to create a formula for https://www.lets-talk-about.tech/2018/03/display-images-in-line-with-terminal.html

I tested it and it works fine, but there is not git repo (just copy and paste instructions).

Is there an example with similar software?

**Comment by SMillerDev** [comment, marked as answer]
That does not meet the requirements we have for homebrew-core, but you could create a formula in your own tap where you use `(bin/"file").write`

**Comment by GETandSELECT** [comment]
> That does not meet the requirements we have for homebrew-core

why?

**Reply by SMillerDev** [reply]
https://docs.brew.sh/Acceptable-Formulae#stable versions

And 

https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff

Are the main rules that come to mind.
